### DISCUSSION TITLE
TypeScript Explorer

### DISCUSSION THREAD

**Discussion by mxsdev** [microsoft/vscode-discussions#402] [Extension Show and Tell]
<p align="center">
<img alt="demo" src="https://user-images.githubusercontent.com/16108792/200933940-b735a2a3-cc9d-40de-a4a9-c10c080eead8.gif" />
</p>

[TypeScript Explorer][gh] is a plugin I made to help get type information in TS projects. It helps for situations where VSCode's quick info (hover box) doesn't give enough depth. [Get it here!][marketplace]

If you find it useful, a star is greatly appreciated. It is still also in preview/beta mode, and bug reports/PRs are welcomed.

Hope some of you find this useful!

[gh]: https://github.com/mxsdev/ts-type-explorer
[marketplace]: https://marketplace.visualstudio.com/items?itemName=mxsdev.typescript-explorer&ssr=false

**Comment by isidorn** [comment]
Very cool, thanks for sharing. fyi @danielrosenwasser
